### PR TITLE
DBZ-3100 Vitess Connector supports Vitess 9.0.0

### PR DESCRIPTION
**Pull request by keweishang**
https://issues.redhat.com/browse/DBZ-3100

Related PRs:

- Vitess Connector Documentation: https://github.com/debezium/debezium/pull/2121
- Vitess tutorial: https://github.com/debezium/debezium-examples/pull/156

### CODE REVIEW

**Comment by keweishang** [comment]
Rendered page:
image

@gunnarmorling Do you think something like this would work?

File: _includes/release.html [hunk 1]
```diff
@@ -131,6 +131,9 @@ <h3 class="section">Tested Versions</h3>
                   {% else %}
                     <span class="test-with-subcategory"> JDBC Driver: </span> {{ driverVersions  | join: ", "  }} <br />
                   {% endif %}
+                  {% if integration.name == 'Vitess' %}
```
Comment (gunnarmorling) [inline _includes/release.html:134]: How about checking for the "note" value being non-empty instead? That way, it should work for other connectors, too, and we'd avoid the empty "*:" for Vitess 1.4.

**Review by gunnarmorling — COMMENTED** [review]
@keweishang, yes, that's pretty much what I was thinking of, thanks! One small suggestion inline.

File: _includes/release.html (hunk 1 above)
Comment (keweishang) [reply _includes/release.html]: Great suggestion! Addressed it in the code. Now the rendered page looks like the following:
image

File: _includes/release.html [hunk 2]
```diff
@@ -131,6 +131,9 @@ <h3 class="section">Tested Versions</h3>
                   {% else %}
                     <span class="test-with-subcategory"> JDBC Driver: </span> {{ driverVersions  | join: ", "  }} <br />
                   {% endif %}
+                  {% if compat.note != null %}
+                    <span class="test-with-subcategory"> *: {{ compat.note  }} </span> <br />
```
Comment (gunnarmorling) [inline _includes/release.html:135]:
```suggestion
                    <span class="test-with-subcategory"> * {{ compat.note  }} </span> <br />
```

**Review by gunnarmorling — APPROVED** [review]

**Comment by gunnarmorling** [comment]
Excellent, it's perfect now. Thank you so much, @keweishang!

**Comment by gunnarmorling** [comment]
Will be live on the website in a few min.

**Comment by keweishang** [comment]
Excellent! Thanks a lot @gunnarmorling!

**Pull request merged**